### DISCUSSION TITLE
I have transferred ethereum from coinbase and it has not arrived in my wallet

### DISCUSSION THREAD

**Discussion by javieralvarezvega** [blockscout/blockscout#9231] [General]
Hello.
Yesterday I transferred ethereum to my Ledger nano s plus wallet and I have not received anything.
At Coinbase they tell me that everything is correct and they sent me a documentation with all the data.
Greetings.

Hola.
Ayer hice un trasaso de ethereum a mi wallet Ledger nano s plus y no he recibido nada.
En Coinbase me dicen que por su parte está todo correcto y me enviaron una documentacion con todos los datos.
Saludos.

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
